packages/cosmos-bin/src/main.rs

```diff
@@ -47,6 +47,10 @@ struct Opt {
     /// Turn on verbose output
     #[clap(long, short, global = true)]
     verbose: bool,
+
+    /// Gas multiplier for simulated transactions
+    #[clap(long, short, global = true, default_value = "1.3")]
```

Comment:
Not a big deal, but it may make more sense to set this as `Option<f64>` and leave off the `default_value`, instead using the default within the library already. Also, it may be easier for your usage if you add `env = "COSMOS_GAS_MULTIPLIER"` so you can simply set that env var during a session.